cpm

```diff
@@ -71,6 +74,18 @@ case "$1" in
             exit 1
         fi
         ;;
+    f)
+        case $2 in
+            /*) OP='from';;
+            *)  OP='files';;
```

Comment:
I am wondering if it makes sense to test with `if [ -e "$2" ]` or something here, rather than just a leading /. For example, I might do this:
```
cd /usr/share/man/man1
cfm f bash.1
```

The way it is now, this would not have the intended result. However, I could also see an argument for the opposite: if you were in /usr/bin and did `cpm f zsh` then it is ambiguous. You may have wanted to see the contents of the zsh package, but you may have wanted to see what packages the file "zsh" was from.

Reply:
Personally I would say you should be doing absolute paths anyway.  

Maybe I can do `starts-with /* || file-exists "$2"`?

Reply:
crappy pseudocode since im on windows atm

Reply:
Yeah, that makes sense, I think. In a normal situation I doubt there would be many cases where you'd be doing `cpm f <packagename>` when `packagename` is a file in the current directory anyway.